packages/site/src/content/Emphasis/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Emphasis, Text } from "@jobber/components";
+import EmphasisContent from "@atlantis/docs/components/Emphasis/Emphasis.stories.mdx";
+import Props from "./Emphasis.props.json";
+import { ContentExport } from "../../types/content";
+
+export default {
+  content: () => <EmphasisContent />,
+  props: Props,
+  component: {
+    element: Emphasis,
+    defaultProps: {
+      children: (
+        <Text>
+          To <Emphasis variation="bold">boldly</Emphasis> go…
+        </Text>
```

Comment:
@scotttjob Looking for your suggestion/preference

if I don't put the component in the children variable, I can do something like:
```
export const Emphasis = (
  props: PropsWithChildren<{
    readonly variation: "bold" | "italic" | "highlight";
  }>,
) => {
  return (
    <Text>
      To <EmphasisRoot {...props}>boldly</EmphasisRoot> go…
    </Text>
  );
};

export default {
  content: () => <EmphasisContent />,
  props: Props,
  component: {
    element: Emphasis,
    defaultProps: {
      variation: "bold",
    },
  },
  ```
  
  But then, imo, the code tab makes a bit less sense because it doesn't represent the actual code in the example:
  
![Screenshot 2024-11-26 at 5 56 30 PM](https://github.com/user-attachments/assets/5bc92521-3b04-485d-ba7e-40c1edf293bc)

Do you have a preference? Does this matter right now?

Reply:
Doesn't matter in a huge way no, Once we have the components in the system we can double back and make a list of the ones that need a bit of additional attention.

Feel free to try out different solution though, and see if something sticks! If we find something that works and makes sense we can quickly move back through the existing entries to make the change and then update the vscode snippet.